README.md

```diff
@@ -55,7 +55,7 @@ Additional system requirements:
 * Inside of your copy, feel free to modify any file you like! It's your copy!
 * Feel free to change any of the default options in `init.lua` to better suit your needs.
 * For adding plugins, there are 3 primary options:
-  * Add new configuration in `lua/custom/plugins/*` files, which will be auto sourced using `lazy.nvim`
+  * Add new configuration in `lua/custom/plugins/*` files, which will be auto sourced using `lazy.nvim` (uncomment line 206 in init.lua therefore)
```

Comment:
Maintaining a specific line number can be challenging, as it requires constant monitoring of every change to `inin.lut` at line 206

Reply:
Agreed. Please describe the line to be removed rather than citing a line number and I'll merge this.

Also please consider changing "therefore" to something like 'to enable this'.

Reply:
Yeah you're right, I changed the hint according to your suggestions